crates/runner-host/src/main.rs

```diff
@@ -79,7 +81,11 @@ async fn main() -> Result<()> {
                 }
             }
         });
+        ensure_trunk()?;
         let url = format!("{}:{}", flags.web_options.web_host, flags.web_options.web_port);
+        let mut build_command = Command::new("trunk");
```

Comment:
We could/should use the wrapper script for reproducibility and to avoid contaminating the user environment. However, we might want to only use the wrapper script based on some flag in the future (once/if the host runner becomes an independent binary). Or even better, we should use trunk as a library if it's possible, such that the host runner is a self-contained binary.

For this PR, let's keep things like that. Just FYI and future steps.

Reply:
I changed it to used the wrapper script. Thanks for pointing that out. 

And I was hoping to use trunk as a library as well and I looked into it somewhat. From what I could find though there doesn't seem to be a publicly documented api for using trunk like that.

There is an internal module that the CLI uses but idk if there's any guarantees about the stability of that interface. 

See https://github.com/trunk-rs/trunk/blob/main/src/build.rs

If you think it would be worthwhile to look into that though I'd be happy to.

Reply:
Thanks for looking into it. I'm still unsure whether it's worth it. It will be useful for development, but I intend to have the host runner as a standalone released binary (or maybe part of the wasefire CLI), in which case it doesn't really matter because users are not supposed to build the host runner anymore.